UnitTests/GravatarTests/GravatarServiceTests.cs

```diff
@@ -46,19 +46,19 @@ public void Setup()
         [Ignore("Need to abstract WebClient or replace with HttpClient")]
         public void GetAvatarAsync_should_call_gravatar_if_absent_from_cache()
         {
-            //_service.ConfigureCache("", 1);
-            //var avatar = Resources.User;
-            //_cache.GetImageAsync(Arg.Any<string>(), null).Returns(_ => null, _ => avatar);
+            // _service.ConfigureCache("", 1);
+            // var avatar = Resources.User;
+            // _cache.GetImageAsync(Arg.Any<string>(), null).Returns(_ => null, _ => avatar);
 
-            //var image = await _service.GetAvatarAsync(Email, 1, DefaultImageType.Identicon);
+            // var image = await _service.GetAvatarAsync(Email, 1, DefaultImageType.Identicon);
 
-            //image.Should().Be(avatar);
-            //Received.InOrder(async () =>
-            //{
+            // image.Should().Be(avatar);
+            // Received.InOrder(async () =>
+            // {
             //    await _cache.Received(1).GetImageAsync($"{Email}.png", null);
             //    await _cache.Received(1).AddImageAsync($"{Email}.png", Arg.Any<Stream>());
             //    await _cache.Received(1).GetImageAsync($"{Email}.png", null);
-            //});
+            // });
```

Comment:
:bulb: This is code &rarr; `#if false` around the block

:bulb: If the code compiles it can be uncommented as the `[Ignore]` attribute will prevent it from executing.

Reply:
Non compiling.